native-code/android/index.md

```diff
@@ -49,6 +49,16 @@ the code, building etc.
       * To build for 32-bit x86: use `target_cpu="x86"`
       * To build for 64-bit x64: use `target_cpu="x64"`
 
+     **NOTICE: Component Builds are not supported**
```

Comment:
I'm wondering if it's not better to just leave this out entirely and send a PSA: to discuss-webrtc to just clarify that it isn't and has never been supported. This is quite a lot of text added for something we just don't support.